src/darker/git.py

```diff
@@ -130,8 +134,13 @@ def _git_check_output_lines(
     """Log command line, run Git, split stdout to lines, exit with 123 on error"""
     logger.debug("[%s]$ %s", cwd, " ".join(cmd))
     try:
-        return check_output(["git"] + cmd, cwd=str(cwd)).decode("utf-8").splitlines()
+        result = run(["git"] + cmd, cwd=str(cwd), check=True, stdout=PIPE, stderr=PIPE)
+        return result.stdout.decode("utf-8").splitlines()
     except CalledProcessError as exc_info:
+        if exc_info.returncode == 129 and exc_info.stderr.decode("utf-8").startswith(
+            "Not a git repository"
+        ):
+            raise NotGitRespository(str(cwd) + " is not a git repository") from exc_info
```

Comment:
For consistency, I'd use an f-string here like in other corresponding places:
```suggestion
            raise NotGitRespository(f"{cwd} is not a git repository") from exc_info
```